### PR TITLE
Convert Mock Digital Twin to be compatible with standard adapters

### LINKED ISSUE #142: [enhancement] Standard Freyja Application
## Description

Implement a standard Freyja binary application that serves as the "default" version of Freyja

## Acceptance criteria

- [ ] The `freyja` library has a binary target that integrates all of the standard adapters together
- [ ] This binary should support managed subscribe in a manner similar to Ibeji

### LINKED ISSUE #143: [enhancement] Integrate mock digital twin with standard adapters
## Description

The mock digital twin should be integrated with standard adapters to facilitate their use.

## Acceptance criteria

- [ ] Mock digital twin is integrated with standard digital twin adapter (requires #124)
- [ ] Old adapters relating to HTTP are deleted

### PR DESCRIPTION
**Pull request by wilyle**
- Converts the Mock Digital Twin to be compatible with the standard adapters (fixes #143)
- Creates the standard Freyja application since it was convenient for testing (fixes #142)
- Removes any HTTP-based adapters that were used with the old implementation as well as deprecated examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,24 @@ This guide uses `apt` as the package manager in the examples. You may need to su
 
 ### Using Freyja
 
-To use Freyja, you will need to implement some adapters and write the main executable that will run the Freyja application.
+Freyja supports a default runtime that's integrated with a set of standard adapters. To build and run the Standard Freyja Runtime, run the following command:
```
Comment (ashbeitz) [inline README.md:41]: nit: It's best not to use contractions like "that's" in professional writing.

File: docs/tutorials/custom-adapters.md [hunk 2]
```diff
@@ -25,6 +23,40 @@ freyja = { git = "https://github.com/eclipse-ibeji/freyja" }
 tokio = { version = "1.0", features = ["macros"] }
 ```
 
-In most cases, the `main.rs` file can be implemented using the `freyja_main!` macro which will take care of writing some boilerplate code for you. This macro only needs adapter type names as input and will generate the main function signature and body. For an example of how to use this macro, see the code for the [in-memory example](../../freyja/examples/in-memory.rs) or the [mock example](../../freyja/examples/mocks.rs).
+In most cases, the `main.rs` file can be implemented using the `freyja_main!` macro which will take care of writing some boilerplate code for you. This macro only needs adapter type names as input and will generate the main function signature and body. For an example of how to use this macro, see the code for the [Standard Freyja Runtime](../../freyja/src/main.rs).
```
Comment (ashbeitz) [inline docs/tutorials/custom-adapters.md:26]: nit: Insert a comma between "macro and "which".

File: docs/tutorials/custom-adapters.md [hunk 3]
```diff
@@ -25,6 +23,40 @@ freyja = { git = "https://github.com/eclipse-ibeji/freyja" }
...
+- [Mock Mapping Service Adapter](../../adapters/mapping/mock_mapping_service_adapter/README.md): Communicates with the [Mock Mapping Service](../../mocks/mock_mapping_service/README.md), which is an executable that mocks a Mapping Service. The behavior is very similar to the in-memory mock, but the application is interactive and allows users to add or remove mappings by pressing enter to advance through configurable states.
+- [gRPC Mapping Adapter](../../adapters/mapping/grpc_mapping_adapter/README.md): Communicates with a mapping service that implements the [`MappingService` protobuf service](../../interfaces/mapping_service/v1/mapping_service.proto). This is a "standard adapter" that is suitable for use in production scenarios.
+
+### Cloud Adapters
+
+- [In-Memory Mock Cloud Adapter](../../adapters/cloud/in_memory_mock_cloud_adapter/README.md): Emulates a Cloud Connector entirely within the memory of the Freyja application. When data is emitted to this adapter it will be printed to the console window.
```
Comment (ashbeitz) [inline docs/tutorials/custom-adapters.md:49]: Nit: Insert a comma between "adapter" and "it".

**Review by ashbeitz — COMMENTED** [review]
Several suggestions. Please take a look.

File: docs/tutorials/custom-adapters.md (hunk 3 above)
Comment (wilyle) [reply docs/tutorials/custom-adapters.md:49]: I think it's grammatically correct as is, but there's a simpler way to write this that I prefer

**Review by ashbeitz — APPROVED** [review]
Approved.

**Pull request merged**